src/libs/OptionsListUtils.js

```diff
@@ -349,7 +349,7 @@ function createOption(logins, personalDetails, report, reportActions = {}, {
     result.text = reportName;
     result.subtitle = subtitle;
     result.participantsList = personalDetailList;
-    result.icons = ReportUtils.getIcons(report, personalDetails, policies, personalDetail.avatar);
+    result.icons = ReportUtils.getIcons(report, personalDetails, policies);
```

Comment:
NAB what about making this:

```suggestion
    result.icons = ReportUtils.getIcons(report, personalDetails, policies, personalDetail && personalDetail.avatar);
```

Reply:
Or changing `const personalDetail = personalDetailList[0];` above to `const personalDetail = personalDetailList[0]; || {}`?

Reply:
The `getIcons` function falls back to a default icon anyway, and the param has a default. But yeah, that might have been better... Maybe you could submit a follow up if the test passes.

Reply:
Submitted a PR for this [here](https://github.com/Expensify/App/pull/11022), because it was found to create a [new blocker](https://github.com/Expensify/App/issues/11019).